src/lib/mod.rs

```diff
@@ -71,3 +71,70 @@ const FASTQ_EXTENSIONS: [&str; 2] = ["fastq", "fq"];
 pub fn is_fastq_path<P: AsRef<Path>>(p: &P) -> bool {
     is_path_with_extension(p, FASTQ_EXTENSIONS)
 }
+
+// Tests
+#[cfg(test)]
+pub mod tests {
+    use crate::*;
+    use rstest::rstest;
+    use seq_io::fastq::OwnedRecord;
+    use std::str;
+    use tempfile::TempDir;
+
+    /// Helper function takes a sequence and returns a seq_io::fastq::OwnedRecord
+    ///
+    fn write_owned_record(seq: &str) -> OwnedRecord {
+        let read = OwnedRecord {
+            head: ("@Sample").as_bytes().to_vec(),
+            seq: seq.as_bytes().to_vec(),
+            qual: vec![b'X'; seq.len()],
+        };
+        read
+    }
+
+    // ############################################################################################
+    // Tests reverse_complement()
+    // ############################################################################################
+
+    #[test]
+    fn test_reverse_complement() {
+        let read = write_owned_record("ACTG");
+        let result = reverse_complement(read.seq);
```

Comment:
why do we need to build an `OwnedRecord` when we're just passing in `read.seq`?  Can't we just create `[u8]`?
```suggestion
        let result = reverse_complement(b"ACTG");
```
May have to add a `to_vec` or something?